src/bin/console/console.cpp

```diff
@@ -1078,6 +1086,58 @@ bool Console::_handle_rollback() {
   return false;
 }
 
+bool Console::_verify_plan(const std::vector<std::shared_ptr<AbstractLQPNode>>& lqp) {
+  const auto& sm = StorageManager::get();
+
+  for (const auto& node : lqp) {
```

Comment:
Please add a short comment why you're only looking at the root nodes here ("Create/Drop can only appear at the root")